src/main/java/lotto/step1/Validation.java

```diff
@@ -0,0 +1,19 @@
+package lotto.step1;
+
+
+public class Validation {
+    private final String text;
+
+    public Validation(String text) {
+        this.text = text;
+    }
+
+    public boolean checkEmptyAndNull() {
+        return text.isEmpty() || text == null;
```

Comment:
```suggestion
        return text == null ||  text.isEmpty();
```
로 변경하면 NPE 문제를 회피 할 수 있어요.

Reply:
헉... 이렇게 간단한 문제를 .. 감사합니다 ^^